### DISCUSSION TITLE
请问有没有对应的web ui

### DISCUSSION THREAD

**Discussion by busgo** [bfenetworks/bfe#825] [Q&A (问题和回答)] [answered]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by iyangsj** [comment]
感谢你的关注！BFE开源项目计划在H2发布控制平台

**Comment by iyangsj** [comment]
Duplicate of #801

**Comment by busgo** [comment]
> Duplicate of #801

麻烦请更新下 https://www.bfe-networks.net/zh_cn/  里面相关内容 有些内容都没有

**Reply by clarinette9** [reply]
已更新https://www.bfe-networks.net/zh_cn/

**Comment by mileszhang2016** [comment, marked as answer]
BFE的控制面已开源发布，见下面的说明
https://github.com/bfenetworks/api-server
